src/codeocean/computation.py

```diff
@@ -141,20 +141,37 @@ def run_capsule(self, run_params: RunParams) -> Computation:
 
         return Computation.from_dict(res.json())
 
-    def wait_until_completed(self, computation: Computation, polling_interval: int = 5) -> Computation:
+    def wait_until_completed(
+        self, computation: Computation, polling_interval: float = 5, timeout: float | None = None,
```

Comment:
To be consistent with argument formatting elsewhere in the package - if not in a single line then arg per line:
```suggestion
        self,
        computation: Computation,
        polling_interval: float = 5,
        timeout: Optional[float] = None,
```

Reply:
Let's also please apply the same in the rest of the PR